inspirehep/modules/workflows/tasks/actions.py

```diff
@@ -1139,3 +1140,15 @@ def check_if_france_in_raw_affiliations(obj, eng):
     for aff in chain.from_iterable(raw_affs):
         if "france" in aff.lower():
             return True
+
+
+def load_record_from_hep(obj, wf):
```

Comment:
do we use it only in core selection workflow?

Reply:
Yes, I think so.